Dockerfile.dev

```
@@ -1,44 +1,53 @@
+WORKDIR /app
+
+# Optional build-time placeholders for Better Auth / Discord (runtime env from compose).
+ARG BETTER_AUTH_URL=http://localhost:3000
+ARG CLIENT_ID=000000000000000000
+ENV BETTER_AUTH_URL=${BETTER_AUTH_URL}
+ENV CLIENT_ID=${CLIENT_ID}
+
+# Install yt-dlp and its dependencies (always latest from pip)
+# Also install build tools needed for native Node.js modules
+# Use pipx to install yt-dlp to avoid PEP 668 issues (pipx puts binaries in ~/.local/bin, not on PATH).
+RUN apk add --no-cache python3 py3-pip ffmpeg build-base autoconf automake libtool g++ pipx su-exec wget \
```

Hadolint (2.14.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)